appservice/src/createAppService/createAppService.ts

```diff
@@ -79,3 +128,20 @@ export async function createAppService(
 
     return wizardContext.site;
 }
+
+function setDefaultsForImprovedConfiguration(wizardContext: IAppServiceWizardContext, fsPath: string): void {
+    const files: string[] = find.fileSync(fsPath);
+    for (const file of files) {
+        if (file.indexOf('package.json') >= 0) {
```

Comment:
Should always use platform built-in functions for file manipulation.  Too many edge cases.

var path = require('path');
path.name('index.html'); // I think (?)

Reply:
Also that would keep you from matching mypackage.json and package.json.txt etc.